docs/scos/dev/front-end-development/202307.0/oryx/building-applications/styling/oryx-responsive-design.md

```diff
@@ -0,0 +1,215 @@
+
+Oryx provides five t-shirt size values, that are given by a generic `size` enumeration, that can be imported from the [utilities package](https://www.npmjs.com/package/@spryker-oryx/utilities).
+
+| Screen      | Code      | Example devices                  |
+| ----------- | --------- | -------------------------------- |
+| Extra small | `Size.xs` | Small phone, Smart Watch         |
+| Small       | `Size.sm` | Smart phone, instore application |
+| Medium      | `Size.md` | Tablet                           |
+| Large       | `Size.lg` | Desktop                          |
+| Extra large | `Size.xl` | Wide screen                      |
+
+While the `xs` and `xl` sizes are available in the typescript enumeration, they are not used in most Oryx components and layouts. You can however use the definitions to create an optimized user experience for these screen sizes. Moreover, you can introduce additional screen sizes in case you need.
+
+The actual screen definition is provided by a breakpoint configuration and can be referenced by their code. If you need to use the screen definitions in your custom code, you can use the `ScreenService`, provided by the [layout package](https://www.npmjs.com/package/@spryker-oryx/layout).
+
+{% info_block infoBox "" %}
+
+Screen sizes might be confusing in some cases. For example, when you browser an Oryx application on a wide screen in a smaller browser window, the physical screen size is not relevant, as the browser viewport is used to indicate the screen size.
+
+{% endinfo_block %}
+
+## Breakpoints
+
+Breakpoints play a crucial role in responsive design, as they define the specific screen sizes at which your website or application layout should adapt. Oryx's Responsive Design system provides default breakpoint for small, medium and large screen sizes. You can adjust the defaults or introduce custom breakpoints to tailor the design system to your specific needs.
+
+Breakpoints are configurable in themes. If you use an Oryx theme it comes with preconfigured breakpoints, but you can override the breakpoints in an additional theme configuration.
+
+{% info_block infoBox "" %}
+
+You'd assume that breakpoints and their usage would be controlled by design tokens, and used as CSS variables. Unfortunately, CSS variables can not be used inside media queries definitions. This is the main reason why Oryx provides a configurable system to provide the breakpoint definitions.
+
+{% endinfo_block %}
+
+### Default Breakpoints
+
+Oryx provides the following default breakpoints:
+
+| Breakpoint | Size Range (in pixels) |
+| ---------- | ---------------------- |
+| `Size.sm`  | Up to 767              |
+| `Size.md`  | 768 - 1023             |
+| `Size.lg`  | 1024 and above         |
```

Comment:
```suggestion
| `Size.Sm`  | Up to 767              |
| `Size.Md`  | 768 - 1023             |
| `Size.Lg`  | 1024 and above         |
```

Reply:
Changed the table a bit for better readibility.